### ISSUE TITLE
Full Screen Routes within CupertinoTabView

### ISSUE THREAD

**Issue by adriancmurray** [flutter/flutter#18541]
I have a navigation question. I want to implement a full screen page route that resides within a CupertinoTabView. I'd like to have my collection of these kinds of routes but have the option to be full screen for a few of them. I've tried all sorts of options but none have worked yet. Any help is appreciated.

```
CupertinoTabView(
routes: <String, WidgetBuilder>{
// Insert routes for using the Navigator from within tabbed page.
},

builder: (BuildContext context)
{
switch (index)
{
case 0:
return page1;
break;
case 1:
return page2;
break;
case 2:
return page3;
break;
case 3:
return page4;
break;
default:
}
},
)
```

**Comment by zoechi** [comment]
@Hixie

**Comment by adriancmurray** [comment]
Thanks for tagging him, @zoechi 

I'd like to clarify something real quick, the CupertinoTabView that I'm using is apart of a custom class I've made to implement material design (plus some other widgets I've made based on material widgets) with iOS human interface guidelines. That's why I'd like to have some routes in full screen from this widget and would like to be able to reference these routes from anywhere within my application like a normal named route. My custom widget resides in a MaterialApp widget, so maybe there is a way I could reference it's routes?
